<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {
background-color: rgb(18, 18, 18);
border: 3px solid rgb(239, 205, 129);
}

QDateEdit {
background-color: rgb(18, 18, 18);
color: rgb(239, 205, 129);
font: 20px;
border: 0px solid;
}

QPushButton {
background-color: rgb(237, 201, 125);
}

QPushButton:hover {
background-color: rgb(247, 222, 153);
}

QToolButton {
background-color: rgb(237, 201, 125);
}

QToolButton:hover {
background-color: rgb(247, 222, 153);
}
</string>
  </property>
  <widget class="QDateEdit" name="FechaInicial">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(18, 18, 18);
color: rgb(239, 205, 129);
selection-background-color: rgb(207, 175, 110);
selection-color: rgb(20, 20, 20);
alternate-background-color: rgb(76, 76, 76);</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="FechaFinal">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(18, 18, 18);
color: rgb(239, 205, 129);
selection-background-color: rgb(207, 175, 110);
selection-color: rgb(20, 20, 20);
alternate-background-color: rgb(76, 76, 76);
</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#edc97d;&quot;&gt;FECHA INICIAL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#edc97d;&quot;&gt;FECHA FINAL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="BotonVolver">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>440</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 30px;
</string>
   </property>
   <property name="text">
    <string>тна</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BotonImprimir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 35 10pt &quot;Cambria&quot;;

</string>
   </property>
   <property name="text">
    <string>IMPRIMIR</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ListaClientes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>441</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 205, 129);
font: 20px;</string>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
